.github/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
```

Comment:
- [ ] Kindly ensure that your ignore is not in your `.github` folder but instead in your `root` folder. The project root is the folder which is the parent for all the project sources